specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json

```diff
@@ -0,0 +1,644 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v2.1",
+    "contact": {
+      "name": "Microsoft Cognitive Services",
+      "url": "https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/",
+      "email": "mlapi@microsoft.com"
+    },
+    "title": "Text Analytics Client",
+    "description": "The Text Analytics API is a suite of text analytics web services built with best-in-class Microsoft machine learning algorithms. The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction and language detection. No training data is needed to use this API; just bring your text data. This API uses advanced natural language processing techniques to deliver best in class predictions. Further documentation can be found in https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview"
+  },
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/text/analytics/v2.1",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+    "paths": {
+        "/languages": {
+            "post": {
+              "tags": [
+                "Languages"
+              ],
+                "description": "The API returns the detected language and a numeric score between 0 and 1. Scores close to 1 indicate 100% certainty that the identified language is true. A total of 120 languages are supported.",
+                "operationId": "56f30ceeeda5650db055a3c7",
```

Comment:
`operationId` should "{operationGroupName}_{operationName}" better keep consistent with previous versions.

Reply:
Made it consistent with previous versions